docs/tutorials/python-development-environment.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Setting up a Python development environment
+subject: Software
+---
+By Anton Vanhoucke
+
+* Table of Contents
+{:toc}
+
+This tutorial shows how to set up a nice working environment for developing Python on Ev3dev. The platform used is a BrickPi, but most of it is usable for regular Lego Ev3 bricks.
+
+Ev3dev is a lean and modern system to run on the raspberry pi that powers the brickpi. It's got multiple languages that work in a pretty similar way and make switching easy. Moreover, if you code in ev3dev on the brickpi, people with a regular Ev3 and Ev3dev will be able to run your code! Alas, it doesn't work the other way around as the BrickPi has a more limited set of motor commands.
+
+For this tutorial, I will suppose you have burned an SD disk, connected your RPi and BrickPi to an ethernet cable and booted it. If you have an Ev3 brick I without ethernet, [set up another networking connection first]().
+
```

Comment:
I don't the the "I" is supposed to be in "Ev3 brick I without" Also, need a url for the link here.